x-pack/legacy/plugins/apm/server/lib/transaction_groups/fetcher.ts

```diff
@@ -60,19 +69,18 @@ export function transactionGroupsFetcher(
         }
       },
       aggs: {
-        transactions: {
-          terms: {
-            ...projection.body.aggs.transactions.terms,
-            order: { sum: 'desc' as const },
-            size: config['xpack.apm.ui.transactionGroupBucketSize']
+        compositeTransactions: {
```

Comment:
nit: I think `transaction_groups` would be a better name here, no need to use the aggregation type.